Mathlib/Geometry/Manifold/ContMDiff.lean

```diff
@@ -2030,6 +2030,83 @@ nonrec theorem Smooth.smul {f : M → 𝕜} {g : M → V} (hf : Smooth I 𝓘(
   hf.smul hg
 #align smooth.smul Smooth.smul
 
+/-! ### Smoothness of an open embedding and its inverse -/
```

Comment:
Can you pull pull this into a section? Then, you can add a variables line containing e.g.
`{M : Type*} [TopologicalSpace M] [Nonempty M] {e : M → H} (h : OpenEmbedding e) {n : WithTop ℕ}`,
prettifying this even more :-)